doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc

```diff
@@ -0,0 +1,34 @@
+[id='test-designer-create-test-template-dmn-based-proc']
+= Creating a test scenario template for DMN-based test scenarios
+
+Create a test scenario template for DMN-based scenarios by following the procedure below to validate your rules and data.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project for which you want to create the test scenario.
+. Click *Add Asset* -> *Test Scenario*.
+. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets have been assigned or will be assigned.
+. Select *DMN* as the *Source type*.
+. Select an existing DMN asset using the *Choose DMN asset* option.
+. Click *Ok* to create and open the test scenario in the *Test Scenario* designer.
+. To map the *GIVEN* column header to a data object,
```

Comment:
Minor suggestion - replace comma with colon:

s/. To map the *GIVEN* column header to a data object,/. To map the *GIVEN* column header to a data object:

Reply:
Changed as suggested.